code/datums/components/storage/storage.dm

```diff
@@ -624,6 +624,10 @@
 			if(!stop_messages)
 				to_chat(M, "<span class='warning'>[IP] cannot hold [I] as it's a storage item of the same size!</span>")
 			return FALSE //To prevent the stacking of same sized storage items.
+	if(HAS_TRAIT(I, TRAIT_NO_STORAGE))
+		if(!stop_messages)
+			to_chat(M, "<span class='warning'>\the [I] can't seem to fit in \the [host]! You're not sure it would fit in any container at all!</span>")
```

Comment:
I am VERY open to suggestions here. Something that clearly and unambiguously tells the player "this thing does not go in backpacks" without breaking being meta/immersion breaking.